platform_umbrella/apps/home_base/lib/home_base/accounts.ex

```diff
@@ -65,6 +65,10 @@ defmodule HomeBase.Accounts do
     Repo.all(User)
   end
 
+  def delete_user(%User{} = user) do
+    Repo.soft_delete(user)
+  end
```

Comment:
Should we simply https://github.com/revelrylabs/ecto_soft_delete/tree/master?tab=readme-ov-file#repos

`use Ecto.SoftDelete.Repo` in `HomeBase.context/0`  or someplace like that?